database/postgres/index.js

```diff
@@ -0,0 +1,109 @@
+const pool = require('./db');
+
+exports.getRestaurantReviews = (restaurantId) => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // same as what was being sent to db before
+      .query('SELECT  FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
+
+exports.getSortedRestaurantReviews = () => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // same as above but sorted
+      .query('SELECT * FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
+
+exports.addOneReview =() => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // needs to return one review from the id
+      .query('SELECT * FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
+
+exports.getOneReview =() => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // needs to return one review from the id
+      .query('SELECT * FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
+
+exports.updateOneReview =() => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // needs to return one review from the id
+      .query('SELECT * FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
+
+exports.deleteOneReview =() => {
+  pool
+  .connect()
+  .then(client => {
+    return client
+      // needs to return one review from the id
+      .query('SELECT * FROM users WHERE id = $1', [1])
+      .then(res => {
+        client.release()
+        console.log(res.rows[0])
+      })
+      .catch(err => {
+        client.release()
+        console.log(err.stack)
+      })
+  })
+};
```

Comment:
Digging the functions to help your endpoints. I'm wondering if there's a way to connect in general so you're not consistently repeating code.

Reply:
That's a great point.  If I end up moving forward with psql, I will definitely take that to heart and try to simplify it.